azkaban-hadoop-security-plugin/src/main/java/azkaban/security/AbstractHadoopSecurityManager.java

```diff
@@ -401,12 +401,22 @@ protected void doPrefetch(final File tokenFile, final Props props, final Logger
     final Credentials cred = new Credentials();
 
     try {
+      // cred is being populated
       fetchAllHadoopTokens(userToProxyFQN, userToProxy, props, logger, cred);
       getProxiedUser(userToProxyFQN).doAs((PrivilegedExceptionAction<Void>) () -> {
         registerAllCustomCredentials(userToProxy, props, cred, logger);
         return null;
       });
+
+      logger.info("fetched cred = " + cred);
```

Comment:
[Logging] any strong reasons we shouldn't keep this as a `debug` level? I have the same question for other logging too.